Makefile

```diff
@@ -1,7 +1,8 @@
 # Image URL to use all building/pushing image targets
 REGISTRY ?= quay.io
+REPO ?= kubevirt
```

Comment:
Can you please move Make file changes to another commit?

Reply:
This bit is needed, the use case for external is something like this 

```bash
export KUBECONFIG=....
export KUBEVIRT_PROVIDER=external
export DEV_REGISTRY=docker.io
export REPO=quiquell
make cluster-sync
```
So we need to parameterize REPO since is going to be the intermediate repository between user and cluster.

Reply:
I meant to the all the changes in  Makefile so it will be easier to review :)